src/System.Net.Http/tests/FunctionalTests/PlatformHandlerTest.cs

```diff
@@ -10,7 +10,8 @@
 
 namespace System.Net.Http.Functional.Tests
 {
-
+    // Test only WinHttpHandler since the curl handler was remove
```

Comment:
Nit: search and replace "was remove" => "was removed"

Reply:
Nit: I would prefer the sentence read "Test only WinHttpHandler since CurlHandler was removed."

Reply:
Fixed